src/chainparams.cpp

```diff
@@ -174,6 +174,13 @@ class CMainParams : public CChainParams {
         consensus.vDeployments[Consensus::DEPLOYMENT_BIP147].nWindowSize = 4032;
         consensus.vDeployments[Consensus::DEPLOYMENT_BIP147].nThreshold = 3226; // 80% of 4032
 
+        // Deployment of InstantSend autolocks
+        consensus.vDeployments[Consensus::DEPLOYMENT_ISAUTOLOCKS].bit = 3;
+        consensus.vDeployments[Consensus::DEPLOYMENT_ISAUTOLOCKS].nStartTime = 1533945600; // Aug 11th, 2018
+        consensus.vDeployments[Consensus::DEPLOYMENT_ISAUTOLOCKS].nTimeout = 1565481600; // Aug 11th, 2019
```

Comment:
Why are we using timestamps instead of block height?

Reply:
This means of deploying changes (version bits) is defined by [BIP-9](https://github.com/bitcoin/bips/blob/master/bip-0009.mediawiki) which uses median time past. I am not sure why the authors chose to use time vs height, but that is how we got here. :slightly_smiling_face: